br/backup-and-restore-overview.md

```diff
@@ -119,15 +119,15 @@ Backup and restore might go wrong when some TiDB features are enabled or disable
 
 ### Version compatibility
 
-Before performing backup and restore, BR compares and checks the TiDB cluster version with its own. If there is a major-version mismatch, BR prompts a reminder to exit. To forcibly skip the version check, you can set `--check-requirements=false`. Note that skipping the version check might introduce incompatibility.
-
 > **Note:**
 >
-> When performing backup and restore, it is recommended that the BR and TiDB cluster are of the same version.
+> It is recommended to use the BR of the same major version as your TiDB cluster for backup and restoration.
+
+Before performing backup and restore, BR compares the TiDB cluster version with its own and checks their compatibility. If the versions are incompatible, BR reports an error and exits. To forcibly skip the version check, you can set `--check-requirements=false`. Note that skipping the version check might introduce incompatibility in data.
 
```

Comment:
It seems that the content in line 130~132 in https://github.com/pingcap/docs-cn/pull/15012/files# is missed. Please check and update. Thanks!